Data_Transfer_Hash/readme.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: pattern
+title: data transfer hash
+folder: Data_Transfer_Hash
+permalink: /patterns/Data_Transfer_Hash/
+categories: Architectural
+tags:
+ - Performance
+---
+
+## Also known as
+
+* data transfer hash
+
+## Intent
+
+Providing a  class called data transfer hash, in order to send message between other class.
+
+## Explanation
+
+
+> Data transfer hashes may be implemented as simple hash tables (or HashMaps). 
+A more robust implementation uses a container object to hold the hash, as well as identifying information, type-safe data retrieval methods, and well-known keys.
+> https://www.oreilly.com/library/view/j2ee-design-patterns/0596004273/re12.html

```

Comment:
Add the link to another section, see https://github.com/iluwatar/java-design-patterns/wiki/02.-Pattern-template